client/src/components/mapping/TabColumnPicking/StaticValueForm/index.tsx

```diff
@@ -155,7 +277,7 @@ const StaticValueForm = ({ attribute }: Props) => {
       </div>
       <FormGroup labelFor="text-input" inline={true}>
         <ControlGroup>
-          {attribute.isReferenceType ? (
+          {attribute.definition.id === 'Reference.type' ? (
```

Comment:
too many ternaries, can you find an other way to render it ?

Reply:
Would it be better with ifs and elses?
I have these 3 cases so I don't think I have many alternatives...

Reply:
`attribute.definition.id === 'Reference.type' && renderMyComponent()`
That should hold in one line and is more readable in my opinion

Reply:
If I do this for the 3 cases, it won't be clear that only 1 of them will be rendered, I think

Reply:
Do you find it clearer now?

Reply:
yes!